orm.go

```diff
@@ -709,6 +693,31 @@ func createAttributesString(attrs map[string]interface{}) (string, error) {
 	return strings.Join(attrsList, ","), nil
 }
 
+func formatIDKey(idKey interface{}) (string, error) {
+	switch v := idKey.(type) {
+	case uint64:
+		return strconv.FormatUint(v, 10), nil
+	case string:
+		return fmt.Sprintf(`'%s'`, v), nil
+	case int:
```

Comment:
we may as well support all integer types 32bit and above, something like:

`case int,int32,int64,uint,uint32,uint64`